pkg/apiserver/resthandler.go

```diff
@@ -589,6 +589,7 @@ func patchResource(ctx api.Context, admit updateAdmissionFunc, timeout time.Dura
 				return nil, err
 			}
 			if hasConflicts {
+				glog.V(4).Infof("patchResource failed for resource %s, becauase there is a meaningful conflict.\n diff1=%v\n, diff2=%v\n", name, diff1, diff2)
```

Comment:
I can pass in the gvk as a input parameter to patchResource, but it seems a little redundant. Your call.